python/cudf/cudf/core/frame.py

```diff
@@ -339,12 +339,7 @@ def memory_usage(self, deep=False):
         -------
         The total bytes used.
         """
-        if deep:
-            warnings.warn(
-                "The deep parameter is ignored and is only included "
-                "for pandas compatibility."
-            )
-        return {name: col.memory_usage for name, col in self._data.items()}
+        raise NotImplementedError
```

Comment:
Why is this not implemented now?

Reply:
Ok so a user is unlikely to run into this error since they would be using one of these other public facing objects. Is that right?

Reply:
Yup, that's correct.